components/ai_chat/resources/page/components/assistant_response/index.tsx

```diff
@@ -67,6 +67,11 @@ export default function AssistantResponse(props: { entry: mojom.ConversationTurn
           <div className={styles.searchInProgress}><ProgressRing />Improving answer with Brave Search…</div>
         )
       }
+      if (event.pageContentRefineEvent && props.isEntryInProgress && !hasCompletionStarted) {
+        return (
+          <div className={styles.searchInProgress}><ProgressRing />{getLocale('pageContentRefinedInProgress')}</div>
```

Comment:
@petemill should we also change the `searchInProgress` style name to be something more general?

Reply:
changed that to `actionInProgress` in 0b4ee1e1e116f7b2224ed02c99dc5198de4677c7